src/components/basic/DlPanelContainer/DlPanelContainer.vue

```diff
@@ -185,6 +189,10 @@ export default defineComponent({
         showCollapseOnHover: {
             type: Boolean,
             default: false
+        },
+        isStudioLeftPanel: {

```

Comment:
Since this is a common component, have a less specific name.
The name should describe what it's doing

Reply:
Since these changes are specific for Studio panel hence I selected this name for the prop.
Could you suggest some variable name?

Reply:
You can call it `layout-mode' and have it as a string. Default to `regular` and have a mode of `studio`